package.json

```diff
@@ -70,8 +70,9 @@
   "browserslist": [
     "> 5%",
     "last 2 versions",
-    "ie > 8",
-    "Safari >= 8"
+    "ie > 10",
```

Comment:
Sorry to be annoying but I've just noticed that you can remove this line and the same browsers will be targeted.

Reply:
😡Done 😉